tests/storage/reservation.go

```diff
@@ -39,7 +39,7 @@ import (
 // feature gate PersistentReservation. The enablement/disablement of this
 // feature gate redeploys virt-handler pod, and this might interfer with other
 // tests.
-var _ = SIGDescribe("[Serial]SCSI persistent reservation", Serial, func() {
+var _ = SIGDescribe("[Serial]SCSI persistent reservation", Label("kvmanualreset"), Serial, Ordered, func() {
```

Comment:
What is the biggest time consumer here?

Reply:
rerolling the kubevirt installation after enabling/disabling the persistent reservation feature gate

Reply:
So I would be inclined to enable the feature gate for Kubevirt testsuite as whole and remove the enabling here.

Reply:
yeah me too.. talking to the ginkgo maintainer this "ordered" container thing is not really advised

Reply:
>So I would be inclined to enable the feature gate for Kubevirt testsuite as whole and remove the enabling here.

with your blessing I will pivot to doing that @alicefr @xpivarc

Reply:
@xpivarc we still need to add a configuration for this feature, but in general persistent reservation should always been off by default since it deploys an additional privileged container. Right now, we are wrongly using the fg for enabling/disabling the feature